earth_enterprise/src/fusion/autoingest/plugins.src/PacketGen.src

```diff
@@ -120,22 +120,19 @@ void PacketGenAssetVersionImplD::UpdateChildren(
     prevTime = curTime;
 
     // Determine which insets intersect us at this level
-    const khLevelCoverage levCov(config.coverage.levelCoverage(level));
+    khLevelCoverage levCov(config.coverage.levelCoverage(level));
     std::vector<uint> neededIndexes;
+
     neededIndexes.reserve(numInsets);
-    if (type == AssetDefs::Imagery) {
-      FindNeededImageryInsets(khInsetCoverage(levCov),
-                              extra.insetInfo, numInsets,
-                              neededIndexes,
-                              config.beginMinifyLevel,
-                              config.endMinifyLevel);
-    } else {
-      FindNeededTerrainInsets(khInsetCoverage(levCov),
-                              extra.insetInfo, numInsets,
-                              neededIndexes,
-                              config.beginMinifyLevel,
-                              config.endMinifyLevel);
-    }
+    auto gencov = khInsetCoverage(levCov);
+    //overlapEnvelope<ProductAssetVersion> env(type, gencov,
+    //                     extra.insetInfo, extra.insetInfo.size(),
+    //                     config.beginMinifyLevel, config.endMinifyLevel, level);
+
+    extra.calc->env.gencov = gencov;
```

Comment:
Since `extra.calc->env.gencov` is a `khInsetCoverage&`, it seems like `extra.calc->env.gencov = gencov;` is going to end up modifying the object that `extra.calc->env.gencov` references ( `genInfo[i].coverage` from *RasterProject.src) instead of modifying the reference itself.
Not sure if that's what we want to do.

Reply:
Resolved by latest commit.